tox.ini

```diff
@@ -1,12 +1,10 @@
 [tox]
-envlist = py269, py27, py352, py361
+envlist = py27,py36
 
 [testenv]
-setenv =
-    PYTHONPATH = {toxinidir}:{toxinidir}/sceptre
-deps =
-    -r{toxinidir}/requirements_tests.txt
-commands = py.test --basetemp={envtmpdir} --junitxml=build/pytest/junit-{envname}.xml
+deps = -rrequirements.txt
```

Comment:
should this be -r requirements.txt?

Reply:
Not sure why, but it has to be without the space